src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudService.kt

```diff
@@ -520,13 +513,14 @@ class CosmasGoogleCloudService(private val freeBucketName: String,
                               responseObserver: StreamObserver<CosmasProto.ChangeFileIdResponse>) {
         LOG.info("Get request for change files ids in project={}", request.info.projectId)
 
-        val project = this.fileBuffer.getValue(request.info.projectId)
-
-        synchronized(project) {
-            for (change in request.changesList) {
-                if (project.contains(change.oldFileId)) {
-                    project[change.newFileId] = project[change.oldFileId]
-                    project.remove(change.oldFileId)
+        val project = this.fileBuffer[request.info.projectId]
+        if (project != null) {
```

Comment:
Is it possible that project is null? What shall we do in this case?

Reply:
What if we had some history before, but for whatever reason Cumulus restarted. Immediately after restart we ran git sync in Papeeria. Will we report id change here and do nothing?